features/importing/archivist.feature

```diff
@@ -135,3 +185,12 @@ Feature: Archivist bulk imports
     When I check the 1st checkbox with id matching "importing_for_others"
     And I press "Import"
     Then I should see "We have notified the author(s) you imported works for. If any were missed, you can also add co-authors manually."
+
+  Scenario: Update the URL on an imported work with a redirect
+
+  # Given I have an Open Doors committee member "Ariana"
```

Comment:
O.O
:D
However, if this pull request is ready to go, can we not have commented out code in it? It has a habit of lingering and it's generally good practice to reduce comment-induced noise.